orm/src/main/java/org/hibernate/search/v6poc/entity/orm/cfg/IndexingStrategyConfiguration.java

```diff
@@ -34,22 +33,22 @@
 
 	private String externalRepresentation;
 
-	private IndexingMode(String externalRepresentation) {
+	private IndexingStrategyConfiguration(String externalRepresentation) {
 		this.externalRepresentation = externalRepresentation;
 	}
 
 	/**
-	 * Returns the {@link IndexingMode} matching the given external representation as specified via
+	 * Returns the {@link IndexingStrategyConfiguration} matching the given external representation as specified via
 	 * {@link SearchOrmSettings#INDEXING_STRATEGY}
 	 * @param indexingMode the indexing mode external representation
-	 * @return the {@link IndexingMode}
+	 * @return the {@link IndexingStrategyConfiguration}
 	 */
-	public static IndexingMode fromExternalRepresentation(String indexingMode) {
+	public static IndexingStrategyConfiguration fromExternalRepresentation(String indexingMode) {
```

Comment:
Right, I should have done it right away after my PR introducing the multi-tenancy-strategies.